docs/proposals/003-atomic-t.md

```diff
@@ -31,42 +31,70 @@ Proposed Approach
 -----------------
 
 We define an `Atomic<T>` type that functions as a wrapper of `T` and provides atomic operations:
-```
+```csharp
+enum MemoryOrder
+{
+    Relaxed = $(kIRMemoryOrder_Relaxed),
+    SeqCst = $(kIRMemoryOrder_SeqCst),
```

Comment:
FWIW Vulkan doesn't actually support sequentially consistent atomics. We do support AcquireRelease which is close, but doesn't support some obscure use cases. I think it would be fine to only have Relaxed for now.

Reply:
I see. `SeqCst` was added because metal supports it.

Reply:
I guess we can have AcquireRelease here and map it to SeqCst on metal.